specifyweb/frontend/js_src/lib/components/Header/Notifications.tsx

```diff
@@ -124,7 +147,9 @@ export function Notifications({
     return (): void => {
       document.removeEventListener('visibilitychange', handler);
       destructorCalled = true;
-      if (timeout !== undefined) globalThis.clearTimeout(timeout);
+      if (timeout !== undefined) {
```

Comment:
@CarolineDenis @melton-jason can you weight in on whether you prefer single statement `if` to be with `{` `}` or without?

I generally find it's easier to make types and harder to see how to fix a typo when there are a lot of `}` and `)` near each other, so as a general rule try to avoid them as much as possible. That also makes code cleaner, looking more like Python.

Though I could see the benefit of always using `{` `}` as that would reduce the git diff when changing to have one/more than one statement.

Whatever you chose, I would just advise to be consistent with it. We have an eslint rule that removes all redundant  `{` `}`, but that is not running at the moment due to tests failing